src/insert/spacefinder/article-body-adverts.ts

```diff
@@ -147,9 +151,10 @@ const addDesktopInline1 = (): Promise<boolean> => {
 		minAbove: isImmersive ? 700 : 300,
 		minBelow: 300,
 		opponentSelectorRules: {
+			// don't place ads right after a heading
```

Comment:
This seemed a little contradictory before: We allow an ad >5px below a heading but we must keep >190px above a heading? We shouldn't put ads closely below heading as this is likely to break up semantically related content. I think this was perhaps a mistake given the confusing naming of `minAboveSlot `.